build.sbt

```diff
@@ -108,16 +99,9 @@ lazy val commonSettings = Def.settings(
     // Filtering tests that are not stable in Scala 2.13 yet.
     Tests.Argument(TestFrameworks.ScalaTest, "-l", "play.api.libs.json.UnstableInScala213")
   ),
-  headerLicense := {
-    val currentYear = java.time.Year.now(java.time.Clock.systemUTC).getValue
-    Some(
-      HeaderLicense.Custom(
-        s"Copyright (C) 2009-$currentYear Lightbend Inc. <https://www.lightbend.com>"
-      )
-    )
-  },
-  scalaVersion := ScalaVersions.scala212,
-  crossScalaVersions := Seq(ScalaVersions.scala212, ScalaVersions.scala213),
+  headerLicense := Some(HeaderLicense.Custom(s"Copyright (C) 2009-2020 Lightbend Inc. <https://www.lightbend.com>")),
```

Comment:
Good to not depend on java time API.

Btw, in Play and Lagom we followed the Scala project and we removed the years. We only have "Copyright (C) 2009-2020 Lightbend Inc."

Reply:
We can move the `commonSettings` to the `Common` plugin. Also let it depend on the Header plugin, like we did for play-soap.

Reply:
My typo, I meant to remove the year.

Reply:
I'll leave it to keep the delta smaller.